sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1006,7 +1006,7 @@ private void SetNameFieldsIfNull()
             }
 
             // Watch out for exploding Responses
-            long length = response.IsUnavailable() ? 0 : response.Value.Length;
+            long length = response.IsUnavailable() ? 0 : response.Headers.ContentLength ?? 0;
```

Comment:
This is how this code looked before migration:

```
     // Watch out for exploding Responses
            long length = response.IsUnavailable() ? 0 : response.Value.ContentLength;
            Pipeline.LogTrace($"Response: {response.GetRawResponse().Status}, ContentLength: {length}");
```

Reply:
Using response.Value.Length directly would cause exception because the stream is not seekable.